client/src/components/Feed/Post.js

```diff
@@ -122,7 +124,10 @@ const Post = ({ post }) => {
 
   return (
     <PostCard>
```

Comment:
Let's add a border for the post (only on desktop):

current
![Screen Shot 2020-05-11 at 5 10 28 PM](https://user-images.githubusercontent.com/42228386/81624290-a6c6ef00-93aa-11ea-9b2c-b1abd9fc497d.png)


figma
![Screen Shot 2020-05-11 at 5 10 49 PM](https://user-images.githubusercontent.com/42228386/81624217-81d27c00-93aa-11ea-99aa-b9d26cad9db5.png)
![Screen Shot 2020-05-11 at 5 11 05 PM](https://user-images.githubusercontent.com/42228386/81624221-8303a900-93aa-11ea-9c35-5df858771de9.png)

Reply:
The border is already added. Please review which rule is overwriting in your machine, maybe retina rules.
<img width="200" alt="Captura de Pantalla 2020-05-11 a la(s) 10 24 44 p  m" src="https://user-images.githubusercontent.com/877561/81635221-56658680-93d6-11ea-9b78-9e2e0c9586a4.png">

Reply:
That's strange, my rules includes the padding and border-radius:
![Screen Shot 2020-05-11 at 10 48 51 PM](https://user-images.githubusercontent.com/42228386/81643054-a09c3700-93d9-11ea-8fc6-2af7170c6988.png)

When I turn off the border you added, the border is set by a class from ant-design and I can see the border:

![Screen Shot 2020-05-11 at 10 50 03 PM](https://user-images.githubusercontent.com/42228386/81643158-dccf9780-93d9-11ea-9c40-a00a5588afeb.png)

Reply:
The current border size is 0.4 pixels and 50% transparency according to Figma, I will change it to 0.5px/50%.